packages/neuvector/generated-changes/patch/Chart.yaml.patch

```diff
@@ -10,13 +10,13 @@
 +  catalog.cattle.io/os: linux
 +  catalog.cattle.io/permit-os: linux
 +  catalog.cattle.io/provides-gvr: neuvector.com/v1
-+  catalog.cattle.io/rancher-version: '>= 2.6.5-0 < 2.7.0-0'
++  catalog.cattle.io/rancher-version: '>= 2.6.5-0 < 2.8.0-0'
```

Comment:
A 2.7 chart should only be scoped to 2.7. It should not be visible on 2.6.5.

```
'>= 2.7.0-0 < 2.8.0-0'
```

Reply:
I will update it.